java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/LiveDeployment/ServerEditor.java

```diff
@@ -56,11 +56,6 @@ public JToolBar getToolBar()
         _options.setEditable(false);
         _user.setEditable(false);
 
-        _allocatable.setEnabled(false);
```

Comment:
Why did you change `_allocatable` here?

Reply:
This was an accidental change. Will fix. Good catch!